### ISSUE TITLE
Some KISAO terms do not match KISAO pattern

### ISSUE THREAD

**Issue by matthiaskoenig** [SED-ML/KiSAO#97]
The following terms have incorrect ids (the trailing `_` is incorrect):
```
http://www.biomodels.net/kisao/KISAO#_KISAO_0000800
http://www.biomodels.net/kisao/KISAO#_KISAO_0000801
http://www.biomodels.net/kisao/KISAO#_KISAO_0000802
http://www.biomodels.net/kisao/KISAO#_KISAO_0000803
http://www.biomodels.net/kisao/KISAO#_KISAO_0000804
http://www.biomodels.net/kisao/KISAO#_KISAO_0000805
http://www.biomodels.net/kisao/KISAO#_KISAO_0000806
http://www.biomodels.net/kisao/KISAO#_KISAO_0000807
http://www.biomodels.net/kisao/KISAO#_KISAO_0000808
http://www.biomodels.net/kisao/KISAO#_KISAO_0000809
http://www.biomodels.net/kisao/KISAO#_KISAO_0000810
http://www.biomodels.net/kisao/KISAO#_KISAO_0000811
http://www.biomodels.net/kisao/KISAO#_KISAO_0000812
http://www.biomodels.net/kisao/KISAO#_KISAO_0000813
http://www.biomodels.net/kisao/KISAO#_KISAO_0000814
http://www.biomodels.net/kisao/KISAO#_KISAO_0000815
```

Code example:
```{python}
for term_id in kisao._ontology:
    if "KISAO#_KISAO_" in term_id:
        # logger.warning(f"Incorrect kisao id: {term_id}")
        print(term_id)
        continue
    try:
        kisao_id = kisao_ontology.get_normalized_id(term_id.split('#')[1], dialect=IdDialect.kisao)
    except (KeyError, ValueError) as err:
        print(f"Missing entries: {term_id}")
        print(err)
```

**Comment by jonrkarr** [comment]
These are the terms @luciansmith created. These ids need to be corrected.

**Comment by jonrkarr** [comment]
These ids also should have been added sequentially, starting from the greatest existing id.

**Comment by jonrkarr** [comment]
Corrected in 2.21.
